docs/how-to-use-python-gvfs.md

```diff
@@ -0,0 +1,240 @@
+---
+title: How to use Gravitino Virtual File System with Filesets in Python
+slug: /how-to-use-python-gvfs
+license: "Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2."
```

Comment:
The license header should be updated to use ASF one, please refer to other docs.